src/queries/ArchiveModelPlan.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from '@apollo/client';
+
+export default gql`
+  mutation ArchiveModelPlan($id: UUID!, $changes: ModelPlanChanges!) {
```

Comment:
Would it make sense to have the parameter for this just be a `Bool!`? Right now you could technically call `ArchiveModelPlan` with a whole bunch of changes - seems like the naming might be a bit confusing

Either that, or just rename the query to `UpdateModelPlan` and the other one (that currently has that name) to something like `UpdateModelPlanAndBasics`?